content/docs/command-reference/exp/run.md

```diff
@@ -183,25 +197,84 @@ experiment we just ran (`exp-44136`).
 `dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
 file before running an experiment.
 
+<admon type="warn">
+
+Note that `--set-param` doesn't update your `dvc.yaml`. If you are tracking each
+parameter individually, make sure to update your `dvc.yaml` accordingly when
+appending or removing parameters.
```

Comment:
Do you think it would be helpful to link to https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#parameter-dependencies? No strong opinion to add it.

Note: we could probably better call out support for groups of params there.